Makefile.inc

```diff
@@ -16,6 +16,12 @@ MAKEFLAGS += --warn-undefined-variables
 ifndef PLATFORM
 $(error Please define PLATFORM)
 endif
+ifndef ARCH
+$(error Please define ARCH)
```

Comment:
Could we have x86_64 as the default ARCH?

Reply:
let's not ... the default is easy to set in the Makefile.